reactivesocket-cpp/src/ReactiveStreamsCompat.h

```diff
@@ -2,7 +2,7 @@
 
 #pragma once
 
-#include "reactive-streams-cpp/ReactiveStreams.h"
+#include <ReactiveStreams.h>
```

Comment:
Thanks for weighing in @lehecka. If you're used to seeing a subfolder, then that suggests that is the emerging pattern despite precedent in both directions. 

I'm really not concerned about more than one header file on this, since this is an implementation of Reactive-Streams-JVM which only has 4 interfaces, and that's all this is supposed to be.